### ISSUE TITLE
make-demo-unique: Modify the NAMESPACE regex to broader

### ISSUE THREAD

**Issue by sherl0cks** [rht-labs/labs-demo#9]
I need to match keys in params files such as `PIPELINES_NAMESPACE`, not just `NAMESPACE`

**Comment by oybed** [comment]
@sherl0cks yes, this was considered, but landed on a firm requirement on starting with `NAMESPACE*` - as indicated on the other issue, I'd like to discuss to see if we can come up with a better (i.e.: more generic) solution as I'm not 100% happy with how this turned out.
